extensions/rest-raml/README.md

```diff
@@ -0,0 +1,19 @@
+# @loopback/rest-raml
+
```

Comment:
Let's start this extension as an experimental one and add the standard disclaimer:

https://github.com/strongloop/loopback-next/blob/81fab94a16f09da8d6251cef68c33b127861c0d4/extensions/typeorm/README.md#L11-L16